docs/ownership.md

```diff
@@ -42,12 +42,12 @@ for each domain in the domain list:
 
 For the above dataset that will result in the following:  
 
-| domain            | organization_name_en                 | organization_name_fr                       |
-| ----------------- | ------------------------------------ | ------------------------------------------ | 
-| canada.ca         | Shared Services Canada               | Services partagés Canada                   | 
-| open.canada.ca    | Shared Services Canada               | Services partagés Canada                   | 
-| digital.canada.ca | Treasury Board of Canada Secretariat | Secrétariat du Conseil du Trésor du Canada |
-| 2006census.gc.ca  | Government of Canada                 | Gouvernement du Canada                     |
+| domain            | organization_name_en                     | organization_name_fr                       |
+| ----------------- | ---------------------------------------- | ------------------------------------------ | 
+| canada.ca         | Employment and Social Development Canada | Famille, Enfants et Développement social   |
+| open.canada.ca    | Treasury Board of Canada Secretariat     | Secrétariat du Conseil du Trésor du Canada |
```

Comment:
While this may be true from a government organization perspective, this is not the result that would come out of the algo, since `open.canada.ca` isn't present, so it would find the parent `canada.ca` and use it's organization

Reply:
Looking at it, is that even correct? I thought `open.canada.ca` was owned by `Measurement Canada`

Reply:
Yeah.. Missed that.

Open.canada.ca is owned by the open government team within TBS CIO branch.